pkg/lvm/mount.go

```diff
@@ -198,7 +198,11 @@ func MountVolume(vol *apis.LVMVolume, mount *MountInfo, podLVInfo *PodLVInfo) er
 
 	err = FormatAndMountVol(devicePath, mount)
 	if err != nil {
-		return status.Error(codes.Internal, "not able to format and mount the volume")
+		return status.Errorf(
```

Comment:
error code has two Internal desc, `rpc error: code = Internal desc = rpc error: code = Internal desc =`.  Can we just return err from this place https://github.com/openebs/lvm-localpv/blob/master/pkg/driver/agent.go#L162 instead of returniing status.Error(codes.Internal, err.Error()). Sorry for small nit picks.

Reply:
Haaa.. Good one didn't notice that part concentrated only on error message  :)

Reply:
We are doing this in so many places

Reply:
can you let me know the locations?

Reply:
Updated the code path of `NodePublishVolume` to return `rpc error: code = <code id>` only one time

Reply:
@mittachaitu you only need to return error at location https://github.com/openebs/lvm-localpv/blob/master/pkg/driver/agent.go#L162 only. We don't need to change anything else as status error will be return as it is as CSI response.

Reply:
I proceed the other way around i.e handover responsibility to the caller but I can see some limitations with this approach... I will make changes accordingly

Reply:
addressed the comments PTAL